dali/operators/image/resize/resize_op_impl_gpu.h

```diff
@@ -79,6 +79,7 @@ class ResizeOpImplGPU : public ResizeBase<GPUBackend>::Impl {
       kernels::KernelRequirements &req = kmgr_.Setup<Kernel>(mb_idx, ctx, mb.input, param_slice);
       mb.out_shape = req.output_shapes[0].to_static<frame_ndim>();
     }
+    kmgr_.ReserveMaxScratchpad(0);
```

Comment:
Why it is needed here?

Reply:
To avoid reallocation of the scratchpad between minibatches. Since now the reallocation is not syncrhonous (and scratchpad is not stream ordered yet), it caused hard errors. This guarantees that the scratchpad admits the largest minibatch. Actually, this was meant to work like this from the start - I think this line was even there at some point, just dropped during some refactoring.

Reply:
Understoood.